cli/server.go

```diff
@@ -165,18 +166,18 @@ func server() *cobra.Command {
 				accessURL = localURL.String()
 			} else {
 				// If an access URL is specified, always skip tunneling.
-				tunnel = false
+				shouldTunnel = false
 			}
 
 			var (
-				tunnelErrChan          <-chan error
 				ctxTunnel, closeTunnel = context.WithCancel(cmd.Context())
+				tunnel                 = &devtunnel.Tunnel{ErrorChan: make(chan error, 1)}
```

Comment:
It feels odd to initialize this even though it's not running. What do you think about calling this tunnel error chan, then just overwriting it instead?